Support multi dimension values and configurable metricValues on AWS C… (#352)
kedacore/keda-docs: content/docs/2.1/scalers/aws-cloudwatch.md

  metadata:
    # Required: namespace
    namespace: AWS/SQS

>     # Required: Dimension Name - supports multiple dimensionName with ";" i.e. dimensionName: QueueName;QueueName

Reviewer: We should add the same for dimensionValue

Author: Updated. Can you please  take a look once again, @zroubalik ?

>     # Required: Dimension Name - supports multiple dimensionName with ";" i.e. dimensionName: QueueName;QueueName

Reviewer 2: ```suggestion
    # Required: Dimension Name - Supports specifying multiple dimension names by using ";" as a seperator i.e. dimensionName: QueueName;QueueName
```

However, is it required for me to pass the same dimension name twice?

    dimensionName: QueueName

>     # Required: Dimension Value - supports multiple dimensionValue ";" i.e. dimensionValue: queue1;queue2

Reviewer 2: ```suggestion
    # Required: Dimension Value - Supports specifying multiple dimension values by using ";" as a seperator i.e. dimensionValue: queue1;queue2
```

    # Optional: AWS Secret Access Key, can use TriggerAuthentication as well
    awsSecretAccessKeyFromEnv: AWS_SECRET_ACCESS_KEY # default AWS_SECRET_ACCESS_KEY
    identityOwner: pod | operator # Optional. Default: pod

>     # Optional: defaultMetricCollectionTime

Reviewer 2: Can you please provide some docs on these new parameters in our parameter list please?

For me it's not clear what they represent.

Author: Updated Parameter list in the doc. Can you please review, @tomkerkhove ?

> When `identityOwner` set to `operator` - the only requirement is that the Keda operator has the correct IAM permissions on the CloudWatch. Additional Authentication Parameters are not required.

> - `defaultMetricCollectionTime` - How long in the past (seconds) should the scaler check AWS Cloudwatch. Used to define **StartTime** [official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html)

Reviewer 2: ```suggestion
- `defaultMetricCollectionTime` - How long in the past (seconds) should the scaler check AWS Cloudwatch. Used to define **StartTime** ([official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html))
```

> - `defaultMetricStat` - Which statistics metric is going to be used by the query. Used to define **Stat** [official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html)

Reviewer 2: ```suggestion
- `defaultMetricStat` - Which statistics metric is going to be used by the query. Used to define **Stat** ([official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html))
```

> - `defaultMetricStatPeriod` -  Which frequency is going to be used by the related query. Used to define **Period** [official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html)

Reviewer 2: ```suggestion
- `defaultMetricStatPeriod` -  Which frequency is going to be used by the related query. Used to define **Period** ([official documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricStatistics.html))
```
